app/routes/photos/components/GalleryPictureModal.js

```diff
@@ -53,10 +53,12 @@ const OnKeyDownHandler = ({
 }) => (
   useEffect(
     () => (
+      // eslint-disable-next-line no-sequences
```

Comment:
When we have to have this warning, isn't it better to just rewrite the code?

Reply:
I fixed these, but I am unsure of the () => **(func1(), func2())** syntax meaning. Does this first execute func1, and then return func2? Hopefully you can review my suggested solution in my recently created PR. :slightly_smiling_face: